src/core/config.go

```diff
@@ -354,6 +354,7 @@ func DefaultConfiguration() *Configuration {
 	config.Go.PleaseGoTool = "//_please:please_go"
 	config.Go.EmbedTool = "//_please:please_go_embed"
 	config.Python.PexTool = "//_please:please_pex"
+	config.Python.WheelResolverTool = "//_please:wheel_resolver"
```

Comment:
I wonder if we should look at moving the Python stuff to a plugin and doing this there rather than adding to core plz at this point?

Reply:
Oh yes good point. I'll speak to @Tatskaari about how best to go about it.